packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/Update/UpdateNotificationBanner.tsx

```diff
@@ -0,0 +1,134 @@
+import styled from 'styled-components';
+import { Column, ElevationContext, Icon, Row, Text } from '@trezor/components';
+import { borders, Elevation, mapElevationToBackground, spacingsPx, zIndices } from '@trezor/theme';
+import {
+    UpdateStatus,
+    UpdateStatusSuite,
+    UpdateStatusDevice,
+    mapSuiteUpdateToClick,
+    mapDeviceUpdateToClick,
+} from './updateQuickActionTypes';
+import { Translation, TranslationKey } from '../../../../../Translation';
+import { useDispatch } from '../../../../../../../hooks/suite';
```

Comment:
Yes, e.g. `src/hooks/suite`. I see you prefer this option, but since most of the codebase and devs use the absolute paths, it would be better to stick to it for consistency.